src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -96,55 +108,82 @@ void RoverPositionControl::parameters_update(bool force)
 void
 RoverPositionControl::vehicle_control_mode_poll()
 {
-	bool updated;
-	orb_check(_control_mode_sub, &updated);
-
-	if (updated) {
-		orb_copy(ORB_ID(vehicle_control_mode), _control_mode_sub, &_control_mode);
+	if (_control_mode_sub.updated()) {
+		_control_mode_sub.copy(&_control_mode);
 	}
 }
 
 void
 RoverPositionControl::manual_control_setpoint_poll()
 {
-	bool manual_updated;
-	orb_check(_manual_control_setpoint_sub, &manual_updated);
+	if (_control_mode.flag_control_manual_enabled) {
+		if (_manual_control_setpoint_sub.copy(&_manual_control_setpoint)) {
+			float dt = math::constrain(hrt_elapsed_time(&_manual_setpoint_last_called) * 1e-6f,  0.0002f, 0.04f);
 
-	if (manual_updated) {
-		orb_copy(ORB_ID(manual_control_setpoint), _manual_control_setpoint_sub, &_manual_control_setpoint);
-	}
-}
+			if (!_control_mode.flag_control_climb_rate_enabled &&
+			    !_control_mode.flag_control_offboard_enabled) {
 
-void
-RoverPositionControl::position_setpoint_triplet_poll()
-{
-	bool pos_sp_triplet_updated;
-	orb_check(_pos_sp_triplet_sub, &pos_sp_triplet_updated);
+				if (_control_mode.flag_control_attitude_enabled) {
+					// STABILIZED mode generate the attitude setpoint from manual user inputs
+					_att_sp.roll_body = 0.0;
+					_att_sp.pitch_body = 0.0;
+
+					/* reset yaw setpoint to current position if needed */
+					if (_reset_yaw_sp) {
+						const float vehicle_yaw = Eulerf(Quatf(_vehicle_att.q)).psi();
+						_manual_yaw_sp = vehicle_yaw;
+						_reset_yaw_sp = false;
+
+					} else {
+						const float yaw_rate = math::radians(_param_gnd_man_y_max.get());
+						_att_sp.yaw_sp_move_rate = _manual_control_setpoint.r * yaw_rate;
```

Comment:
```suggestion
						_att_sp.yaw_sp_move_rate = _manual_control_setpoint.y * yaw_rate;
```
I think using the roll stick inputs were part of this PR, and need to change for stabilized mode too

Reply:
I was confused as well, but I did fix that yesterday already, but forgot to push - done now.